controllers/submariner/route_agent_resources.go

```diff
@@ -97,7 +85,11 @@ func newRouteAgentDaemonSet(cr *v1alpha1.Submariner, clusterNetwork *network.Clu
 							Path: "/sys",
 						}}},
 						{Name: "host-var-run-openvswitch-nbdb-sock", VolumeSource: corev1.VolumeSource{HostPath: &corev1.HostPathVolumeSource{
-							Path: ovnNBDBPath,
+							Path: "/var/run/openvswitch/ovnnb_db.sock",
+						}}},
+						//Path used by Openshift
```

Comment:
```suggestion
						// Path used by OpenShift
```

Is this really OpenShift-specific, or is it tied to OVN-IC only?

Reply:
it is specific to Openshift IC, since OVN want support certain scenarios when cluster  upgraded, they are using path with ovn-ic instead of OVN , for non-openshift it is just ovn